src/pytest/test_aggregate.py

```diff
@@ -390,6 +390,11 @@ def testReducerGeneratedAliasing(self):
                           'SORTBY', 2, '@brand', 'ASC')
         self.env.assertEqual('__generated_aliasfirst_valuetitle,by,price,desc', rv[1][2])
 
+    def testIssue1125(self):
```

Comment:
Let's also have a test for ft.search that shows that the limit is working there ..